homeassistant/components/velbus/__init__.py

```diff
@@ -169,38 +169,13 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 class VelbusEntity(Entity):
     """Representation of a Velbus entity."""
 
+    _attr_should_poll: bool = False
+
     def __init__(self, channel: VelbusChannel) -> None:
         """Initialize a Velbus entity."""
         self._channel = channel
-
-    @property
-    def unique_id(self) -> str:
-        """Get unique ID."""
-        if (serial := self._channel.get_module_serial()) == "":
-            serial = str(self._channel.get_module_address())
-        return f"{serial}-{self._channel.get_channel_number()}"
-
-    @property
-    def name(self) -> str:
-        """Return the display name of this entity."""
-        return self._channel.get_name()
-
-    @property
-    def should_poll(self) -> bool:
-        """Disable polling."""
-        return False
-
-    async def async_added_to_hass(self) -> None:
-        """Add listener for state changes."""
-        self._channel.on_status_update(self._on_update)
-
-    async def _on_update(self) -> None:
-        self.async_write_ha_state()
-
-    @property
-    def device_info(self) -> DeviceInfo:
-        """Return the device info."""
-        return DeviceInfo(
+        self._attr_name = self._channel.get_name()
```

Comment:
I think (but might need confirmation) that it is recommended to use the argument than the variable in these cases:
```suggestion
        self._attr_name = channel.get_name()
```

This would apply to the whole `__init__` method.